### ISSUE TITLE
Bundle error: `react-native-navigation could not be found within the project`

### ISSUE THREAD

**Issue by stigi** [DataDog/dd-sdk-reactnative#76]
**Describe what happened**
`1.0.0-beta2` introduced a bundle error when the project does not use `react-native-navigation`.

I believe the issue was introduced just recently in #49. It also does not reproduce in `1.0.0-beta1`.

Here's an excerpt from one of our CI runs:

```
error Unable to resolve module react-native-navigation from /home/circleci/project/mobile-app/node_modules/dd-sdk-reactnative/src/rum/instrumentation/DdRumReactNativeNavigationTracking.tsx: react-native-navigation could not be found within the project or in these directories:
  node_modules
  /home/circleci/project/mobile-app/node_modules/react-native-navigation

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
   6 |
Error: Unable to resolve module react-native-navigation from /home/circleci/project/mobile-app/node_modules/dd-sdk-reactnative/src/rum/instrumentation/DdRumReactNativeNavigationTracking.tsx: react-native-navigation could not be found within the project or in these directories:
   6 |
   7 | import React from 'react'
  node_modules
>  8 | import { ComponentDidAppearEvent, Navigation } from 'react-native-navigation';
  /home/circleci/project/mobile-app/node_modules/react-native-navigation
     |                                                      ^
   9 | import { DdRum } from '../../foundation';
  10 |
  11 | /**.
    at ModuleResolver.resolveDependency (/home/circleci/project/mobile-app/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:234:15)
    at DependencyGraph.resolveDependency (/home/circleci/project/mobile-app/node_modules/metro/src/node-haste/DependencyGraph.js:413:43)
    at Object.resolve (/home/circleci/project/mobile-app/node_modules/metro/src/lib/transformHelpers.js:317:42)
    at resolve (/home/circleci/project/mobile-app/node_modules/metro/src/DeltaBundler/traverseDependencies.js:629:33)
    at /home/circleci/project/mobile-app/node_modules/metro/src/DeltaBundler/traverseDependencies.js:645:26
    at Array.reduce (<anonymous>)
    at resolveDependencies (/home/circleci/project/mobile-app/node_modules/metro/src/DeltaBundler/traverseDependencies.js:644:33)
    at /home/circleci/project/mobile-app/node_modules/metro/src/DeltaBundler/traverseDependencies.js:329:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/home/circleci/project/mobile-app/node_modules/metro/src/DeltaBundler/traverseDependencies.js:137:24)
info Run CLI with --verbose flag for more details.
```

**Steps to reproduce the issue:**

Bundle an app with `dd-sdk-reactnative` at version `1.0.0-beta2` and `react-navigation` but without `react-native-navigation`. Make sure to configure the SDK as described in the Readme.

**Describe what you expected:**

No bundle error

**Additional context**
    - Version of the SDK `1.0.0-beta2`
    - ~A copy of your `package.json` file~

**Comment by 0xnm** [comment]
Hello, @stigi! Sorry for the inconvenience. We've just published `1.0.0-beta3`, this should fix the issue you have.

**Comment by stigi** [comment]
Didn't have a chance yet to say thanks for the quick fix 🙌
